### ISSUE TITLE
Document vote widget fixes

### ISSUE THREAD

**Issue by jess-cook03** [mozilla/sumo-project#345]
Logged out:
![Screen Shot 2020-04-14 at 4 47 42 PM](https://user-images.githubusercontent.com/54991698/79272706-d51ae280-7e6f-11ea-972e-80a42ea097e9.png)
Logged in:
![Screen Shot 2020-04-14 at 4 48 12 PM](https://user-images.githubusercontent.com/54991698/79272709-d5b37900-7e6f-11ea-8b81-9f4539ebe07f.png)

Are we able to have non-logged in users provide feedback? Is there a reason we can't do that currently?
Right now -- its showing a blank, no option to provide any feedback.

Secondly -- the UI is a little wonky here and the UX around having an 'x' to close this widget seems unsuccessful now seeing in production. Also the alignment of the checkboxes shouldnt be centered it it should be to the left side.

I think removing the 'x' to close this and allowing all users to provide feedback would be ideal, the goal of this widget is to capture real feedback on the articles from all users.

**Comment by kkellydesign** [comment]
@jess-cook03 
1. The radio buttons are now aligned correctly here: https://github.com/mozilla/kitsune/pull/4312
2. The survey form doesn't care whether the user is logged in or not. Here's an example with the user logged out and the form displaying correctly: 

<img width="320" alt="Screen Shot 2020-04-15 at 1 30 22 PM" src="https://user-images.githubusercontent.com/202590/79368400-4e244380-7f1d-11ea-87df-3d0da41a61aa.png">

The issue is that when you vote once, you're not allowed to vote again too quickly. The server tracks your IP, so whether or not you're logged in or out doesn't matter. There's a line in the backend that says "If user is over the limit, don't save but pretend everything is ok." That's what you get with the "logged out" screenshot above.

So in short, you're only seeing this because you're testing it and triggering server rate limits that are in place in order to prevent spam. When users are trying to use this feature organically, they'll most likely always see the form.

**Comment by ovidiuboca1** [comment]
I verified this issue on desktop (Mac OS X 10.14, Windows 10 with FF and chrome) and mobile (iPhone 8 plus) and I can confirm the fix.

@jess-cook03, @madasan Question: If I'm logged out every time I visit an article the voting widget is displayed (even if I interacted with it) if I'm logged in and I interact with the widget and then refresh the page the voting widget is gone, so the question is if this is expected. From my point of view, I think this is the expected behavior but I wanted to hear your opinion. 
I will move this into "Ready for deploy", please move it back if you think it is necessary.
